tests/PHPStan/Rules/Functions/data/string-replace-subject.php

```diff
@@ -0,0 +1,72 @@
+<?php declare(strict_types=1);
+
+namespace StringReplaceSubject;
+
+class HelloWorld
+{
+	private string $prop = 'Foo';
+
+	public function wrong(string $path): void
+	{
+		$x = strtr('\\', '/', $path);
+		$x = str_replace($path, '\\', '/');
+		$x = preg_replace($path, '\\', '/');
+		$x = str_replace($path, '\\', '/', $count);
+		$x = preg_replace($path, '\\', '/', $count);
+
+		$x = strtr('\\', $path, '/');
+		$x = str_replace('\\', $path, '/');
```

Comment:
looks like shopware is doing such kind of construction:
https://github.com/shopware/shopware/blob/1b0e9dc726a70788a1a99bc7c1bd5939ad54dd30/src/Core/Framework/DataAbstractionLayer/Command/CreateHydratorCommand.php#L389

I feel like shopware should instead better use `sprintf` or something, but it seems we whould not report an error for this situation then...

Reply:
same in laravel/framework: 
- https://github.com/laravel/framework/blob/2b3e8d75f10b0ed17416282946355dc026bf326c/src/Illuminate/View/Compilers/BladeCompiler.php#L466